ext/fg/js/popup.js

```diff
@@ -341,10 +333,9 @@ class Popup {
     }
 
     _invokeApi(action, params={}) {
-        if (!this._isInjectedAndLoaded) {
-            throw new Error('Frame not loaded');
+        if (this._container.contentWindow) {
```

Comment:
This is needed because `Frontend.onSearchClear` calls `Popup.clearAutoPlayTimer` before the popup has initialized. The check could also be in `clearAutoPlayTimer` but it wouldn't make sense without another flag. Could also mean that the call to `onSearchClear` is unnecessary.

Reply:
The call comes from here:
https://github.com/FooSoft/yomichan/blob/af89659751f72ae282f2c81d797cc224d1077b0a/ext/bg/js/search.js#L250